src/components/Header.test.jsx

```diff
@@ -1,26 +1,32 @@
 import React from 'react';
 import { MemoryRouter } from 'react-router-dom';
-import firebase from 'firebase';
 import { render, screen } from '@testing-library/react';
+import { initializeTestApp } from '../lib/testing';
 import Header from './Header';
 
-firebase.initializeApp({ apiKey: 'fake-api-key' });
+initializeTestApp({ apiKey: 'fake-api-key' });
 
-test('does not render alert by default', () => {
+// hacky workaround to allow findBy to work
+// TODO: Figure out a cleaner solution.
+window.MutationObserver = require('mutation-observer');
+
+test('does not render alert by default', async () => {
   delete window.location;
   window.location = new URL('https://www.example.com');
 
   render(<Header />, { wrapper: MemoryRouter });
+  await screen.findByText('Login');
```

Comment:
This is to wait for the auth state to resolve itself and the component to reach its final render state. Without this we get warnings complaining about not handling promise rejections.